CHANGES

```diff
@@ -5,6 +5,7 @@
 - API Change: Net#openURI now returns a boolean to indicate whether the uri was actually opened.
 - API Change: DefaultShader now always combines material and environment attributes
 - API Change: Added ShapeRenderer constructor to pass a custom shader program to ImmediateModeRenderer20.
+- API Change: Added ClasspathFileHandleResolver for resolving assets via Gdx.files.classpath(string).
```

Comment:
Probably better to call the kinds of things API Addition instead of API Change. That said, I'm not sure  it is even necessary to specifically call out this in CHANGES - I guess it doesn't hurt though.